packages/components/index.js

```diff
@@ -27,11 +27,7 @@ export {
 export { PlaceCard } from './src/PlaceCard';
 export { BlackToAlpha };
 export { Shimmer } from './src/Shimmer';
-export { ConnectionCard } from './src/ConnectionCard';
-export type {
-  TripSegmentWithId,
-} from './src/ConnectionCard/ConnectionCardTypes';
-
+export { ItineraryCard, ItineraryCardType } from './src/ItineraryCard';
```

Comment:
`ItineraryCardType` export can be probably removed, it doesn't seem to be used anywhere and is currently causing warning below.

<img width="675" alt="screenshot 2019-02-07 at 16 20 11" src="https://user-images.githubusercontent.com/2660330/52421441-55b7b380-2af4-11e9-8a93-a296510ca47d.png">

Reply:
OK, removed